src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Summary.tsx

```diff
@@ -52,6 +56,15 @@ export function Summary({
           {t(PurchaseOverviewTexts.summary.price(formattedPrice))}
         </ThemeText>
       )}
+      {formattedOriginalPrice && (
```

Comment:
This part for showing original price, if it is available (only shown when price is discounted)